game/screen_sing.cc

```diff
@@ -40,6 +40,11 @@ ScreenSing::ScreenSing(std::string const& name, Audio& audio, Database& database
 	m_selectedTrack(TrackName::LEAD_VOCAL)
 {}
 
+bool ScreenSing::singingDuet() {
+return false;
+// 	return (m_song.hasDuet() && m_duet == 0 && )
+}
+
 void ScreenSing::enter() {
 	keyPressed = false;
 	m_DuetTimeout.setValue(10);
```

Comment:
Duet timeout should be configurable from config

Reply:
What is that timeout even doing? Either way, that magic number was already there. Not my addition.

Reply:
The timeout is there for autoplaying the song if no keyboard is pressed for 10 secs it's autoplaying with the suggested playersetting if i'm right.